tests/dsl.test.ts

```diff
@@ -358,6 +401,44 @@ type T4
 type T5
   relations
     define A as A1 from A2 but not A3
+`);
+      const time2 = new Date();
+      expect(result.length).toEqual(1);
+      expect(time2.getTime() - time1.getTime()).toBeLessThan(1000);
```

Comment:
i'm not 100% sure but i think `it('test name', (....), 1000)` achieves the same goal

Reply:
done